### PR TITLE
Add org.kde.ktimetracker

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://invent.kde.org/pim/ktimetracker/-/issues/1
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* https://invent.kde.org/pim/ktimetracker/-/merge_requests/9

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by Mailaender** [comment]
bot, build org.kde.ktimetracker

**Comment by hfiguiere** [comment]
cc @flathub/kde

**Comment by Mailaender** [comment]
bot, build org.kde.ktimetracker

**Comment by tsdgeos** [comment]
This needs the extra perms so it can minimize itself to the system tray. Otherwise if you open it, close it with the [x] the app is lost forever (in a Plasma environment)

"--talk-name=org.kde.StatusNotifierWatcher"

i think this is the one

**Comment by Mailaender** [comment]
Note: I upped the permissions to IDE level, which might be controversial.

**Comment by Mailaender** [comment]
bot, build org.kde.ktimetracker

**Comment by tsdgeos** [comment]
> Note: I upped the permissions to IDE level, which might be controversial.

Where did you did that?

**Comment by Mailaender** [comment]
I added them to https://github.com/flathub/flathub/pull/3184 by mistake. Sorry, I am getting spammed by notifications here.

**Comment by Mailaender** [comment]
bot, build org.kde.ktimetracker

**Comment by tsdgeos** [comment]
> I added them to #3184 by mistake. Sorry, I am getting spammed by notifications here.

Are you sure you need such permissions here and not in 3184? Why does ktimetracker need so much access?

**Comment by Mailaender** [comment]
Because I confused them. Their names are very similar.

File: org.kde.ktimetracker.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15-21.08",
+  "sdk": "org.kde.Sdk",
+  "command": "ktimetracker",
+  "rename-icon": "ktimetracker",
+  "finish-args": [
+    "--share=ipc",
+    "--device=dri",
+    "--socket=wayland",
+    "--socket=fallback-x11",
```
Comment (hfiguiere) [inline org.kde.ktimetracker.json:12]:
```suggestion
    "--socket=fallback-x11"
```

**Comment by barthalion** [comment]
bot, build org.kde.ktimetracker

**Comment by travier** [comment]
Looks like it need notification / system tray permissions too.

**Comment by Mailaender** [comment]
bot, build org.kde.ktimetracker

**Comment by travier** [comment]
LGTM. Thanks!

**Pull request closed without merging**